### PR TITLE
use cgroups load to modify the cgroup (#2092)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #2092 to release-2.0
You can switch your code base to this Pull Request by using [git-extras](https://github.com/tj/git-extras):
```bash
# In chaos-mesh repo:
git pr https://github.com/chaos-mesh/chaos-mesh/pull/2113
```

After apply modifications, you can push your change to this PR via:
```bash
git push git@github.com:ti-srebot/chaos-mesh.git pr/2113:release-2.0-1a0246eac566
```

---

Signed-off-by: YangKeao <keao.yang@yahoo.com>

### What problem does this PR solve?

Maybe would solve https://github.com/chaos-mesh/chaos-mesh/issues/2048, but not sure and need some test. At least, this PR removes a lot of codes which are nearly the same with `containerd/cgroups`, and make us easier to adapt cgroup v2.

### What is changed and how it works?

What's Changed:

* Use `cgroups.Load(cgroups.Pid())` directly
* Use `YangKeao/cgroups` instead of `containerd/cgroups`

Tests

- [ ] Unit test
- [ ] E2E test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

### CODE REVIEW

**Review by STRRL — APPROVED** [review]
/merge

**Review by cwen0 — APPROVED** [review]
LGTM

File: go.mod [hunk 1]
```diff
@@ -13,12 +13,13 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/ec2 v1.5.0
 	github.com/chaos-mesh/chaos-mesh/api/v1alpha1 v0.0.0
 	github.com/chaos-mesh/k8s_dns_chaos v0.2.0
-	github.com/containerd/cgroups v0.0.0-20200404012852-53ba5634dc0f
+	github.com/containerd/cgroups v1.0.2-0.20210605143700-23b51209bf7b
```
[the remark this answers is not in the document]
Comment (cwen0) [reply go.mod:16]: @sonatype-lift ignore

File: go.mod [hunk 2]
```diff
@@ -73,9 +74,10 @@ require (
 	go.uber.org/fx v1.12.0
 	go.uber.org/zap v1.15.0
 	golang.org/x/lint v0.0.0-20200302205851-738671d3881b // indirect
-	golang.org/x/sync v0.0.0-20201207232520-09787c993a3a
+	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
+	golang.org/x/sys v0.0.0-20210630005230-0f9fa26af87c // indirect
 	golang.org/x/time v0.0.0-20201208040808-7e3f01d25324
-	golang.org/x/tools v0.0.0-20200616195046-dc31b401abb5
+	golang.org/x/tools v0.1.4
```
[the remark this answers is not in the document]
Comment (cwen0) [reply go.mod:80]: @sonatype-lift ignore

**Comment by STRRL** [comment]
e2e test failed caused by unstable network, I will take a retry

**Pull request merged**